src/components/ProjectContributors.js

```diff
@@ -93,10 +94,11 @@ const ProjectContributors = (props) => {
     if (loadingProjectContributors || loadingContributors || loadingGithubContributors) {
         return <LoadingProgress/>
     }
-    if (errorGithubContributors) {
+    const isNotAContributor = () => {
         return (
             <GithubAccessBlocked
-                message={`You must be a Github collaborator to access this metrics`}
+                message={`You must be a Github collaborator to see the other collaborators of this repository.`}
```

Comment:
Change to `You must have access to this repository on GitHub to see an up-to-date list of collaborators`